docs/application/web/guides/data/data-filter.md

```diff
@@ -330,15 +330,15 @@ The following table lists the filter types you can use with specific content att
 
 The following tables list the filter types you can use with specific message attributes in the methods of the [MessageStorage](../../api/latest/device_api/mobile/tizen/messaging.html#MessageStorage) interface.
 
-The following table lists the filter attributes related to the `findMessage()` method.
+The following table lists the attributes by which `Message` objects may be filtered.
 
-**Table: Filter attributes for finding messages**
+<span id="message-filters-table">**Table: Filter attributes for finding Messages**</span>
```

Comment:
When you update this PR, please remove this span tag.  
Only **\*\*Table: Filter attributes for finding Messages\*\*** is enough.

Reply:
Our intention was to make it possible to the tables in this guide from Web API reference.

Currently a hyperlink may only point to _Messaging Filter Attributes_ - it has an _id="messaging-filter-attributes"_ attribute.

Added _<span id="message-filters-table">_  enables linking straight to the tables specific for _Message_ class.

So, can we preserve this _<span>_ tag, to enable linking straight to the table specific for _Message_?

Reply:
If you want to create a link to a point which is not a title, you can use `<a name="message-filters-table"></a>`

For example, if you check web/guides/multimedia/media-controller.md file, 
you'll see "send_custom_commands" is inserted as below to jump to the one of list item.

![image](https://user-images.githubusercontent.com/9958374/52384594-80d4df80-2ac1-11e9-9848-a31d6e8951b4.png)

Would you please use `<a name>` to link to the tables?

Reply:
I have applied the suggested change - anchors are now added with <a name="..."></a>.